datasets/ru/nsd_isin/crawler.py

```diff
@@ -55,7 +48,7 @@ def crawl_item(context: Context, url: str):
 
         if result.type == "date":
             try:
-                value = parse_date(value, context)
+                value = h.extract_date(context.dataset, parse_date(value))[0]
             except ValueError:
```

Comment:
Does this ever raise a value error? Do we want to instead do this when we actually .add the properties, based on looking at `prop.type` ?